pkg/apiserver/server.go

```diff
@@ -87,7 +88,13 @@ func Start(params *APIServerParams) {
 	}
 
 	if !util.IsHelmManaged() {
-		if err := operator.Start(params.AutocreateClusterToken); err != nil {
+		client := &client.Client{
+			TargetNamespace:   util.AppNamespace(),
+			ExistingInformers: map[string]bool{},
+			HookStopChans:     []chan struct{}{},
+		}
+		store := store.GetStore()
```

Comment:
why can't this be [here](https://github.com/replicatedhq/kots/pull/3142/files#diff-99c0290799daf9abc6240df64063e20bfaf67b371577b67ac7eec6f4725622ffR49) and written as:
```go
var operatorStore = store.GetStore()
```